usecases/gitops.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: GitOps
+description: This document describes how Flipt can be used with GitOps workflows
+mode: wide
+---
+
+## Overview
+
+GitOps is a way to do Continuous Delivery, it works by using Git as a single source of truth for declarative infrastructure and applications. With Git at the center of your delivery pipelines, developers can make pull requests to accelerate and simplify application deployments.
+
+Flipt is uniquely suited to work with GitOps workflows because of our declarative backends and ability to be configured completely via YAML.
```

Comment:
```suggestion
Flipt is uniquely suited to work with GitOps workflows because of our declarative backends and ability to be configured completely via YAML, or JSON.
```